mod2/src/main/java/org/example/ParserJson.java

```diff
@@ -0,0 +1,49 @@
+package org.example;
+
+import com.google.gson.*;
+
+
+import java.io.*;
+import java.net.URISyntaxException;
+import java.nio.file.Paths;
+/*
+Application should read a config file on the startup
+Then it should ensure that all of files from the config exist
+Then it should rename each file adding a suffix from the config to its name
+It should print the results of the renaming like:
+old_name -> new_name
+ */
+
+public class ParserJson {
```

Comment:
Guess name is not good. Based on your implementation it looks like this class contains logic to get data from config file and also a logic to rename files from configuration. This is a bit different logic so my advice is to split logic to a 2 different classes i.e. ConfigurationReader and FilesProcessor